template.json

```diff
@@ -100,5 +100,6 @@
       "ts-node": "10.9.1",
       "typescript": "4.6.4"
     }
-  }
+  },
+  "packageManager": "yarn@3.2.3"
```

Comment:
Hmm never seen this before. Couldn't find any documentation for this. What does this do?